site/data/qasida/ya-kitab-al-ghuyub.json

```
@@ -53,6 +57,9 @@
     },
     "translation": {
         "en": [
+            ["The beloved's visage, the ultimate quest,"],
+            ["Who sees, knows, while others are suppressed."],
```

```suggestion
            ["The countenance of the Beloved is the ultimate desire,"]
            ["Whoever sees, understands, while others remain veiled."]
```

I think this is the best that combines what I think is meant here.

Especially the second line where there was variance in how the reference to veiled seems to vary between 'all else' (ie all creation?) or 'others'. My guess is that 'others' is meant in this context for having not got the chance to view the Beloved.

I liked how one had the capitalization of `B` in Beloved so kept that in too

Added the extra punctuation after 'sees' as I think that is how I would say it out loud.